### ISSUE TITLE
[BUG]: A form field element should have an id or name attribute

### ISSUE THREAD

**Issue by irisvw** [fdnd-agency/nebulaxplorer#267]
# Description
A form field element in the footer has neither an id nor a name attribute. This might prevent the browser from correctly autofilling the form.

To fix this issue, add a unique id or name attribute to a form field. This is not strictly needed, but still recommended even if you have an autocomplete attribute on the same element.

**Comment by irisvw** [comment]
The form element has been removed from the footer and rewritten to be on the home page (see #311). I've added both an id and a name to it. 

<img width="651" height="94" alt="Image" src="https://github.com/user-attachments/assets/0775f999-fcd7-4cf4-9f4e-646f6a9df1e4" />

There should still be an autocomplete attribute added to it.
